src/ensembl/src/content/app/browser/Browser.test.tsx

```diff
@@ -84,29 +87,17 @@ describe('<Browser />', () => {
     );
 
   describe('rendering', () => {
-    test('does not render when no activeGenomeId', () => {
+    it('renders an interstitial if no species is selected', () => {
       const { container } = mountBrowserComponent({ activeGenomeId: null });
-      expect(container.innerHTML).toBeFalsy();
+      expect(container.querySelector('.browserInterstitial')).toBeTruthy();
     });
 
-    test('renders links to example objects only if there is no selected focus feature', () => {
-      const { container, rerender } = mountBrowserComponent();
-
-      expect(container.querySelectorAll('.exampleLinks')).toHaveLength(1);
-
-      rerender(
-        <Browser
-          {...defaultProps}
-          browserQueryParams={{
-            focus: faker.lorem.words()
-          }}
-        />
-      );
-
-      expect(container.querySelectorAll('.exampleLinks')).toHaveLength(0);
+    it('renders an interstitial if no feature has been selected', () => {
+      const { container } = mountBrowserComponent();
+      expect(container.querySelector('.browserInterstitial')).toBeTruthy();
     });
 
-    test('renders the genome browser and track panel only when there is a selected focus feature', () => {
+    it('renders the genome browser and track panel only when there is a selected focus feature', () => {
```

Comment:
Why was this changed from `test` to `it`?

Reply:
For consistency with other test files. We sort of decided a while back to use the `it` function name rather than `test`.